test/Microsoft.AspNet.Mvc.Razor.Host.Test/MvcRazorParserTest.cs

```diff
@@ -15,30 +16,132 @@ namespace Microsoft.AspNet.Mvc.Razor
 {
     public class MvcRazorCodeParserTest
```

Comment:
My code doesn't because as far as its concerned it's not its responsibility to catch it. If somehow duplicate tagHelperPrefix directives get to my code it'll all just work. If you feel strongly about the test I can add it (not a lot of work); just didn't want to conflate expectations with how things should work in practice in our tests is all.